src/garage/tf/optimizers/first_order_optimizer.py

```diff
@@ -30,8 +31,8 @@ class FirstOrderOptimizer:
     """
 
     def __init__(self,
-                 tf_optimizer_cls=None,
-                 tf_optimizer_args=None,
+                 optimizer=None,
+                 optimizer_args=None,
```

Comment:
Can you replace `optimizer_args` with `learning_rate`? This is the only parameter we want to tune. This also aligns with your change at `benchmarks/src/garage_benchmarks/experiments/algos/ppo_garage_tf.py`

Reply:
I merged default `optimizer_args` with `tf_optimizer_args` to new `optimizer_args`. Hence, now `optimizer_args` will accept more than `learning_rate` but also other parameters.

Reply:
For easier usage, I replaced `optimizer_args` with `learning_rates` across the codebase